host/windows/syscall.c

```diff
@@ -1488,13 +1490,16 @@ int oe_syscall_uname_ocall(struct oe_utsname* buf)
     // OE SDK is supported only on WindowsServer and Win10
     if (IsWindowsServer())
     {
-        sprintf(buf->sysname, "WindowsServer");
-        sprintf(buf->version, "2016OrAbove");
+        sprintf_s(buf->sysname, strlen("WindowsServer") + 1, "WindowsServer");
+        sprintf_s(buf->version, strlen("2016OrAbove") + 1, "2016OrAbove");
```

Comment:
`sizeof(<string>)` gives the same result (sizeof will include the null terminator) and it will be computed at compile time.